starter-code/css/main.css

```diff
@@ -1,3 +1,188 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: ‘Montserrat’, sans-serif;
+  font-weight: lighter;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px;
+  align-items: center;
+  margin: 10px 15px;
+}
+.navbar img {
+  height: 75px;
+}
+
+h1 {
+  font-size: 5em;
+}
+
+.nav-items {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav-items a {
+  font-size: 20px;
+  text-decoration: none;
+  color: #1a1a1a;
+  margin: 10px;
+}
+
+.landing {
+  background-image: url("/lab-css-spotify-clone/starter-code/images/landing.jpg");
```

Comment:
Here the path should always be relative to the css file.
So: "../images/landing.png